src/Effect/ReadFS.hs

```diff
@@ -107,7 +119,28 @@ deriving instance Ord (ReadFSF a)
 instance ToDiagnostic ReadFSErr where
   renderDiagnostic = \case
     FileReadError path err -> "Error reading file " <> pretty path <> ":" <> line <> indent 4 (pretty err)
-    FileParseError path err -> "Error parsing file " <> pretty path <> ":" <> line <> indent 4 (pretty err)
+    FileParseError path err PythonRequirementErrFmt ->
+      vsep
+        [ "Error parsing file: " <> pretty path <> "."
+        , ""
+        , "We occasionally find files we think are python requirements.txt files but"
+        , "aren't. If this file isn't a python requirements.txt file, this error can"
+        , "be safely ignored."
+        , ""
+        , "Details:"
+        , indent 4 (pretty err)
+        , ""
+        , reportDefectWithFileMsg path
+        ]
```

Comment:
can this message be added as `errCtx` rather than embedding inside of the `FileParseErr`? I'd like to keep these types disjoint if possible

Reply:
Fixed with: https://github.com/fossas/fossa-cli/pull/801/commits/f2ff1d78eb2ebeea819844fa6aca4c7b9f1bc9f9